docker-compose.yml

```diff
@@ -26,6 +26,7 @@ services:
     container_name: civiform
     links:
       - "db:database"
+      # comment below line and set environment variable DISABLE_LOCALSTACK=1 if you don't want localstack
       - "localstack"
```

Comment:
Can we remove this link?  "localstack" should be accessible at that name.  I'd love for DISABLE_LOCALSTACK to be the only thing we need.

Reply:
I think so. The purpose of the link also serves as a declaration of dependency, but we are purposely waiting for localstack to be ready so that is not necessary, at least at the moment. I agreed that only having to set a shell variable is nice, so it seems a good tradeoff.